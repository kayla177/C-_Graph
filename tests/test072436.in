ENTITY 2WEQQ97S name_2WEQ journal
ENTITY KYBGQ86I name_KYBG institution
ENTITY XGXB4LC1 name_XGXB author
RELATIONSHIP 2WEQQ97S affiliated_with KYBGQ86I 3.3
PRINT 2WEQQ97S
HIGHEST
EXIT
