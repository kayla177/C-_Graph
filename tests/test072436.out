success
success
success
success
KYBGQ86I
2WEQQ97S KYBGQ86I 3.3
